docs/protocol/index.md

```diff
@@ -4,55 +4,36 @@
 
-* Follow a core-periphery architecture (such as in Uniswap), which allows for changing out periphery functionality without changing the data models at the core.
-* Allow for extension via modules (such as in MakerDAO).
+* The **On-Chain Registry** anchors Autonolas autonomous services on-chain, and provides the primitives needed to create, operate and secure such services.
 
-Examples of modules include governance and staking. Governance is particularly important in a modular system, as it is used to vote on the adoption or abandoning of modules. By ensuring an immutable core, the Autonolas protocol provides guarantees that once created, the ecosystem’s agent components, canonical agents, and services are not mutable by governance – an important guarantee of censorship resistance.
+* The **Tokenomics** defines economic model facilitated by the Autonolas token. This is a mechanism to grow the capital deployed in services and reward developers.
 
-The Autonolas Protocol is built with the {{open_autonomy}} framework in mind as the primary framework for realizing agent services. However, it does not prescribe the usage of the {{open_autonomy}} framework and allows for services to be implemented on alternative frameworks.
+* The **Governance** defines the rules of Autonolas as a decentralized, autonomous
+organization (DAO), which is governed by the community.
+
+<figure markdown>
+![Autonolas Protocol elements](./images/autonolas_protocol_elements.svg)
+</figure>
 
 !!! abstract "Learn more"
 
     Read the **Technical Architecture**, **Tokenomics** and **Governance** sections in the [Autonolas Whitepaper](https://www.autonolas.network/documents/whitepaper/Whitepaper%20v1.0.pdf) for the full details of the **Autonolas Protocol**.
 
-## Core smart contracts
-
-Core smart contracts are permissionless. Autonolas governance controls the process of service management functionalities and of minting new NFTs representing components and agents (i.e. it can change the minting rules and pause minting). The remaining functionalities, in particular transfer functionalities, are not pausable by governance.
-
-**Generic Registry**
-:	An abstract smart contract for the generic registry template which inherits the Solmate ERC721 implementation. 
-
-**Unit Registry**
-:	An abstract smart contract for generic agents/components template which inherits the Generic Registry
-
-**Component Registry**
-:	An ERC721 contract that inherits the Unit Registry and represents agent components.
+## Rationale
 
-**Agent Registry**
-:	An ERC721 contract that inherits the Unit Registry and represents canonical agents.
+In most settings, the reward model for service owners and agent operators is usually straightforward: users remunerate service owners, and service owners remunerate operators supporting their service. However it is not always well-defined how this is propagated to open software developers. This is where the Autonolas Protocol plays its role.
```

Comment:
I guess this should be "open-source software developers" rather than "open software"?